File: Python/Ex07.py
a= float(input("Digite o primeiro valor: "))
b= float(input("Digite o segundo valor: "))
c= float(input("Digite o terceiro valor: "))
d= float(input("Digite o quarto valor: "))
e= float(input("Digite o quinto valor: "))
soma= a+b+c+d+e
print(f"Total= {soma:.2f}")
pagamento= float(input("Valor pago: "))
troco= pagamento-soma
print(f"Seu troco eh: {troco:.2f}")

File: C/Ex07.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float a, b, c, d, e, soma, pagamento, troco;

    printf("Valor do produto 1: ");
    scanf("%f", &a);
    printf("Valor do produto 2: ");
    scanf("%f", &b);
    printf("Valor do produto 3: ");
    scanf("%f", &c);
    printf("Valor do produto 4: ");
    scanf("%f", &d);
    printf("Valor do produto 5: ");
    scanf("%f", &e);

    soma= a+b+c+d+e;
    printf("Total= %.2f\n", soma);

    printf("Valor pago: ");
    scanf("%f", &pagamento);

    troco= pagamento-soma;

    printf("Seu troco eh: %.2f", troco);
    return 0;

    }



